kyverno/authorization/host-namespaces/disallow-host-ports.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: disallow-host-ports
+  annotations:
+    policies.kyverno.io/description: Provides a list of ranges of allowable ports in the host network namespace.
```

Comment:
@igorkchyts Would change to - `Provides a list of allowed ports in the host network namespace. Access to host ports allows potential snooping of network traffic and should forbidden / minimized.`